lib/sqlalchemy/testing/suite/test_types.py

```diff
@@ -452,6 +453,100 @@ def test_concatenate_clauselist(self, connection):
         )
 
 
+class IntervalTest(_LiteralRoundTripFixture, fixtures.TestBase):
+    __requires__ = ("datetime",)
```

Comment:
we probably need a new require here, called interval or similar

Reply:
Added requires you can review once went with name `datetime_interval`.

While adding only_on, just checked for other DB(s), where i think Interval is supported for arithmetic operations but it is not supported with sqla.

_MSSQL_ : https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/time-date-and-interval-functions?view=sql-server-ver16
_MySQL_ : https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html

Reply:
not sure if there is a reason why they are supported only on pg and oracle. @zzzeek do you remember why?